lib/src/features/authentication/bloc/sign_up_state.dart

```diff
@@ -0,0 +1,44 @@
+import 'package:equatable/equatable.dart';
+
+class SignUpState extends Equatable {
+  final String email;
+  final String username;
+  final String name;
+  final String password;
+  final String confirmPassword;
```

Comment:
Let's validate the email and password. You can refer to [template](https://github.com/GoldenOwlAsia/mobile-flutter-template/blob/development/lib/src/features/authentication/logic/signin_state.dart) and use [formz](https://pub.dev/packages/formz)